FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -467,7 +373,16 @@ TrajectorySeedProducer::produce(edm::Event& e, const edm::EventSetup& es)
             index=  4:   --  -- [FPix1_neg] | hitIndicesInTree[4] (=7)  | trackerRecHits[7]
 
             The implementation has been chosen such that the tree only needs to be build once upon construction.
-        */
+      */
+
+	regions.clear();
+	if(theRegionProducer){
+	  es_ = &es;
```

Comment:
get rid of the variable es_ (also in header)
instead pass es as arugment (const reference) to the function pass2HitsCuts